catalog.tests.bom

```diff
@@ -0,0 +1,164 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+brooklyn.catalog:
+  id: brooklyn-postgresql-node-tests
+  version: 0.10.0-SNAPSHOT
+  itemType: template
+  iconUrl: https://wiki.postgresql.org/images/a/a4/PostgreSQL_logo.3colors.svg
+  name: PostgreSQL Node tests
+  description: Test that PostgreSQL Node is running correctly
+  license_code: Apache-2.0
+  license_url: http://www.apache.org/licenses/LICENSE-2.0.txt
+  item:
+    services:
+    - type: brooklyn-postgresql-node
+      id: postgresqlnode
+      name: PostgreSQL Node
+      brooklyn.config:
+        datastore.creation.script.contents: |
+          CREATE DATABASE temporary;
+          USE temporary;
+          CREATE TABLE users (name VARCHAR(20), owner VARCHAR(20), creation_date DATE, updated_date DATE);
+          INSERT INTO users VALUES ('mysql', 'mysql', CURRENT_DATE, CURRENT_DATE)
+        postgresql.port: 6789
+    - type: org.apache.brooklyn.test.framework.TestCase
+      name: PostgreSQL Node tests
+      targetId: postgresqlnode
+      brooklyn.children:
+
+      # Sensors tests
+      - type: org.apache.brooklyn.test.framework.TestCase
+        name: Sensors Tests
+        brooklyn.children:
+        - type: org.apache.brooklyn.test.framework.TestSensor
+          name: Check PostgreSQL server is up
+          sensor: service.isUp
+          timeout: 10m
```

Comment:
On some clouds, we've seen far longer than 10 minutes to provision the VM and install+launch the entity. Can we make this a configurable value using `brooklyn.parameters`?

I suggest we simplify the timeouts. This is not a performance test. At a minimum, we just need two values: timeout for initial startup; and timeout for anything else. We could therefore declare:

```
    brooklyn.parameters:
    - name: timeout.initialStartup
      description: The timeout for provisioning, installing and launching the app-under-test.
      type: org.apache.brooklyn.util.time.Duration
      default: 20m
    - name: timeout.runtimeAssertion
      type: org.apache.brooklyn.util.time.Duration
      description: The timeout for any other operation (e.g. invoking an effector or waiting for a sensor to be updated)
      default: 5m
```

You could define in the top-level `org.apache.brooklyn.test.framework.TestCase` the timeout to be the "timeout.runtimeAssertion" value. And then only once override that timeout (for the very first "Check PostgreSQL server is up").